bsondump/bsondump.go

```diff
@@ -162,11 +188,11 @@ func (bd *BSONDump) Debug() (int, error) {
 		if doc == nil {
 			break
 		}
-		result.Data = doc
+		result = doc
```

Comment:
Is there a reason we need this result variable instead of calling bson.Unmarshal(doc, &validated)?

Reply:
Whoops - this was on an outdated version. Resolving.